sfs/time/source.py

```diff
@@ -4,6 +4,23 @@
 
 .. include:: math-definitions.rst
 
+.. plot::
+    :context: reset
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import sfs
+
+    plt.rcParams['figure.figsize'] = 6, 6  # inches
```

Comment:
This doesn't work too great with the second plot, because it's not quadratic.

What about removing this line and using `figsize=[6, 6]` for creating the quadratic plot?

Or alternatively, what about making all the plots (i.e. both plots) more or less the same shape?

Reply:
#100 should improve this a bit, but it's still a good idea to use a `figsize` that is appropriate for the shape of the plot.

Reply:
Now the examples in `time.source` are plotted in rectangles,
the same as in `mono.source`.

I think squares look better for SFS examples in `time.drivingfunction`,
as we are using a circular array.